scripts/image_epouta_constants.sh

```diff
@@ -7,6 +7,6 @@ IMAGE_KEEP_COUNT="25"
 TMP_IMAGE_NAME="_tmp_$IMAGE_NAME"
 SECURITY_GROUPS="default"
 TEST_INSTANCE_NAME="_test_$IMAGE_NAME-$(date +%F)"
-TEST_FLAVOR="hpc.mini"
+TEST_FLAVOR="standard.tiny"
```

Comment:
The instance creation test currently uses "hpc.4.10core". 

In cPouta, imgbuild test flavor == instance creation flavor. 

Do we care for similar consistency in ePouta?

Reply:
In that case I think we should change instance creation test to use the standard one because that to me seems less likely to require a change in the future.

Reply:
Works for me. There's >1 nodes in that aggregate so instance creation is not affected by hardware failure in either case.